packages/components/src/inputs/src/InputGroup.module.css

```diff
@@ -78,23 +91,17 @@
 
 .hop-InputGroup[data-disabled] {
     --background: var(--hop-InputGroup-background-disabled);
-    --border: 0.0625rem solid var(--hop-InputGroup-border-color-disabled)
+    --border: 0.0625rem solid var(--hop-InputGroup-border-color-disabled);
+    --input-placeholder-color: var(--hop-InputGroup-placeholder-color-disabled);
 }
 
-.hop-InputGroup__input {
-    --input-font-family: var(--hop-InputGroup-input-md-font-family);
-    --input-font-size: var(--hop-InputGroup-input-md-font-size);
-    --input-font-weight: var(--hop-InputGroup-input-md-font-weight);
-    --input-line-height: var(--hop-InputGroup-input-md-line-height);
-    --input-block-size: var(--hop-InputGroup-input-md-block-size);
-    --input-inline-size: initial;
-    --input-placeholder-color: var(--hop-InputGroup-placeholder-color);
-
+:where(.hop-InputGroup__input),
```

Comment:
Is there a reason for the where? I like using it on basic elements like text or icon but at this level I fell something is off or I need more explanation as to why? Are we expecting overrides?

Reply:
because input group is used inside each field. It is kind a bade for fields. but some fields such as NumberField or TextArea are done a bit differently and don't follow the same style.

Reply:
They could have their added classes such as `.hop-TextArea__input-group`

Reply:
so you're saying you rather do `.hop-TextArea .hop-TextArea__InputGroup`?